### ISSUE TITLE
Is it really necessary to tie compatibility to current ReSharper version?

### ISSUE THREAD

**Issue by astrohart** [testcop/Resharper.TestCop#62]
I _love_ TestCop!  It's my favorite ReSharper extension.

However, that being said, the JetBrains folks like to do Continuous-Release of point releases.  i have the JetBrains Toolbox app on my computer which constantly keeps ReSharper up to date.

However, as soon as ReSharper puts out a new version, testcop stops working.  I've raised this issue before, but I am certain that there has to be a solution so that TestCop can "keep up" with R#.

**Comment by testcop** [comment]
Please see https://github.com/testcop/Resharper.TestCop/issues/43#issuecomment-568255225

Also, I no longer code in C# so I need TestCop fans to prompt me about a recompile...(which I am more than happy to do). Like you...I think testcop is an essential tool..

**Comment by Mike-E-angelo** [comment]
LOL @testcop I was going to post that link earlier but didn't want to be an obnoxious naglord... well, not more than I usually am. 😆

**Comment by testcop** [comment]
Just to be sure, @astrohart ... can I confirm you're able to install TestCop on the latest R# ? 

Also, have any of you tried TestLinker ? Which is a jetbrains maintained plugin? I see they give me a shout out in their tags :)
![image](https://user-images.githubusercontent.com/28064257/107155128-a5911580-696e-11eb-9ac9-da8fefe194e0.png)

**Comment by Mike-E-angelo** [comment]
Yes I have @testcop.  I ran into a few issues as captured in https://github.com/matkoch/resharper-testlinker/issues/10

It also appears the project has been archived. 😟

**Comment by rklec** [comment]
One cannot install/find it anymore with recent Resharper versions as it seems:
![grafik](https://user-images.githubusercontent.com/71322635/165556303-e7bc33b8-dec5-4ee1-9ce6-c5cc67219398.png)

JetBrains ReSharper 2022.1 Build 221.0.20220418.193503 built on 2022-04-18
ReSharper 2022.1.20220418.200314
